.github/workflows/build.yaml

```diff
@@ -96,9 +96,6 @@ jobs:
             name=${{ env.REGISTRY }}/${{ env.REPO_LOWER }}
           tags: |
             type=raw,value=latest,enable={{is_default_branch}}
-            type=sha,enable=true,priority=100,prefix=,suffix=,format=short
```

Comment:
These need to get pulled. Their short sha is 7 digits versus 8. Instead down where we tag the build, we add the short sha, that we know works. And we leave the metadata section for the tag to latest retag action.